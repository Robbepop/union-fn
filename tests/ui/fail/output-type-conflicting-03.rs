fn main() {}

#[union_fn::union_fn]
trait Foo {
    type Output = i32;
    
    fn foo() -> Self::Output {}

    type Output = i64;
}
