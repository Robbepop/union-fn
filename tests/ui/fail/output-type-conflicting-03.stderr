error: encountered conflicting Output types in #[union_fn] trait
 --> tests/ui/fail/output-type-conflicting-03.rs:9:5
  |
9 |     type Output = i64;
  |     ^^^^^^^^^^^^^^^^^^

error: previous Output definition here
 --> tests/ui/fail/output-type-conflicting-03.rs:5:5
  |
5 |     type Output = i32;
  |     ^^^^^^^^^^^^^^^^^^
